saleor/tests/fixtures.py

```diff
@@ -459,6 +459,18 @@ def checkout_with_payments(checkout):
     return checkout
 
 
+@pytest.fixture
+def checkout_with_item_and_preorder_item(
+    checkout_with_item, product, preorder_variant_channel_threshold
+):
+    checkout_info = fetch_checkout_info(
+        checkout_with_item, [], [], get_plugins_manager()
+    )
+    add_variant_to_checkout(checkout_info, preorder_variant_channel_threshold, 1)
+    checkout_with_item.save()
```

Comment:
This `save` on `checkout` is not needed.

Reply:
Nice catch, thanks :)